### ISSUE TITLE
Re-serializers already serialized strings

### ISSUE THREAD

**Issue by manuranga** [wso2/msf4j#154]
I have a already created JSON string. But if I try to return that from a MS the framework re-serializes it.
eg:
```java
String json = "{\"a\":\"b\"}";
return Response.ok(json, "application/json");
```

```
expected = {"a":"b"}
actual   = "{\"a\":\"b\"}"
```

Is there a workaround to get the expected?

**Comment by afkham** [comment]
This is how to get the expected:

Remove @Produces annotation from the service method.

Then in your method, do the following:

```java
String json = "{\"a\":\"b\"}";
JsonParser parser = new JsonParser();
JsonObject obj = new JsonParser().parse(json);

return Response.status(Response.Status.OK).entity(obj).header("Content-Type", MediaType.APPLICATION_JSON).build()
```

**Comment by afkham** [comment]
This has been properly fixed now and will be available in the 2.0 release
